doc/installation.rst

```diff
@@ -90,6 +90,14 @@ To install ``python-can`` using the XL Driver Library as the backend:
 
 3. Use Vector Hardware Configuration to assign a channel to your application.
 
+CANtact
+~~~~~~~
+
+To install ``python-can`` using the CANtact driver backend:
+
+1. Install the Python ``cantact`` package: ``python3 -m pip install cantact``
```

Comment:
Remove the `1.` if there is no second step. It might be worth mentioning that you can install via `pip install "python-can[cantact]"`. You could also provide a link to the cantact docs